projects/admin/src/app/circulation/items.ts

```diff
@@ -176,3 +177,12 @@ export class Item {
     return (this.pending_loans && this.pending_loans.length > 0);
   }
 }
+
+export interface ItemRecordPermission extends RecordPermission {
```

Comment:
add doc block for explaining the interface.

Reply:
as this interface is no more used, I delete it